model/subscriber.go

```diff
@@ -64,3 +70,68 @@ func (s *Subscriber) Copy(dst datastore.Entity) (datastore.Entity, error) {
 func (s *Subscriber) GetCache() datastore.Cache {
 	return nil
 }
+
+// CreateSubscriber creates a new subscriber from the passed subscriber (s).
```

Comment:
Why ```subscriber (s)``` ? It seems like you can only provide 1 ?

Reply:
The bracket is just indicating the variable name